sdk/appconfiguration/azure-data-appconfiguration/README.md

```diff
@@ -223,14 +223,15 @@ String url = configurationClient.getConfigurationSetting(urlKey, urlLabel).getVa
 Connection conn;
 try {
     conn = DriverManager.getConnection(url);
+    conn.close();
```

Comment:
I'd put this in a try-finally block, so if any errors happen, it'll always try to dispose of the connection.
```java
try {
} finally {
    if (conn != null) {
        conn.close();
    }
}
```

Reply:
but how do you handle the exception? Ignore the exception and throws it? like the example below: 
```
 public void sqlExample() throws SQLException {
        ConfigurationClient configurationClient = new ConfigurationClientBuilder()
            .connectionString(connectionString)
            .buildClient();

        // urlLabel is optional
        String url = configurationClient.getConfigurationSetting(urlKey, urlLabel).getValue();
        Connection conn = null;
        try {
            conn = DriverManager.getConnection(url);
        } finally {
            if (conn != null) {
                conn.close();
            }
        }
    }

```

Reply:
Try finally can have a catch block too.
https://www.javatpoint.com/finally-block-in-exception-handling

Reply:
conn.close() throws exception as well